cilium/rockcraft.yaml

```diff
@@ -81,6 +89,9 @@ parts:
       git rev-parse HEAD >SOURCE_VERSION
       make bazel-bin/cilium-envoy
       make install
+    override-prime: |
+      craftctl default
+      rm -rf /root/.cache/bazel
```

Comment:
`cilium-envoy` uses `bazel` to build, and bazel creates a huge cache that's ~`6G`. This doesn't work well with rockcraft especially in CI scenarios, where the rockcraft cache isn't persisted for long. We save a lot of space in the lxd container. I'll actually move this to `override-build` to save up on maximum disk space usage during runtime since `override-prime` will wait for everything to finish.